DRAFT_homogeneous_impact_areas.Rmd

```diff
@@ -0,0 +1,426 @@
+---
+title: "-"
+output:
+  html_document
+editor_options: 
+  markdown: 
+    wrap: sentence
+---
+
+# Homogeneous Impact Areas
+
+<br />
+
+*Author and date:* 
+
+Anders Kolstad
+
+```{r, echo=FALSE}
+Sys.Date()
+```
+
+<br />
+
+<!-- Load all you dependencies here -->
+
+```{r setup, include=FALSE}
+library(knitr)
+knitr::opts_chunk$set(echo = TRUE)
+library(sf)
+library(tmap)
+library(dplyr)
+library(stars)
+library(ggplot2)
+library(units)
+library(ggpubr)
+library(DT)
+#library(osmplotr)
+```
+
+## Introduction
+
+This chapter documents the creation of a wall-to-wall map of homogeneous impact areas in Norway.
+The map is produces by binning values in the infrastructure index into four discrete categories.
+It is likely to be a good predictor for some indicators, such as *slitasje* and the presence of *alien species*.
+
+## About the underlying data
+
+The infrastructure index is explained [here](https://brage.nina.no/nina-xmlui/handle/11250/2983607).
+It is a wall-to-wall raster over Norway with 100 m resolution.
+Each pixel is given a value along a continuous gradient from 0 to around 15.4, representing the frequency or surrounding cells within 500 m with human infrastructure (houses, roads, ect).
+
+### Representativity in time and space
+
+The infrastructure index is calculated with data that have slightly different dates, but can be said to represent the current status, the period 2010-2020.
```

Comment:
"..., in the period..."

Reply:
Rewritten. The data actually represents the year 2015.